pkg/querier/tenantfederation/merge_queryable.go

```diff
@@ -27,7 +27,7 @@ import (
 // NewQueryable returns a queryable that iterates through all the tenant IDs
 // that are part of the request and aggregates the results from each tenant's
 // Querier by sending of subsequent requests.
-// By setting bypassWithSingleQuerier to true the mergeQuerier gets bypassed
+// By setting bypassWithSingleID to true the mergeQuerier gets bypassed
```

Comment:
@dimitarvdimitrov changing to `bypassWithSingleID` makes sense, since the number of queriers had to be abstracted away (only GEM will need multiple queriers). The doc strings need fixing up. I was waiting for Peter's input first :)